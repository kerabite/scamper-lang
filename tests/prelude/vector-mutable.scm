(define sample-vector (vector "alpha" "beta" "gamma" "delta" "epsilon"))
sample-vector
(vector-set! sample-vector 2 "zeta")
sample-vector
(vector-set! sample-vector 0 "foo")
sample-vector
(vector-set! sample-vector 2 -38.72)
sample-vector
(vector-fill! sample-vector "woot")
sample-vector