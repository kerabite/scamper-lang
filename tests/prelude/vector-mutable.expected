(vector "alpha" "beta" "gamma" "delta" "epsilon")
void
(vector "alpha" "beta" "zeta" "delta" "epsilon")
void
(vector "foo" "beta" "zeta" "delta" "epsilon")
void
(vector "foo" "beta" -38.72 "delta" "epsilon")
void
(vector "woot" "woot" "woot" "woot" "woot")
